README.md

```diff
@@ -812,10 +813,11 @@ See also [Ticketing](#ticketing), [Task management/To-do lists](#task-management
   * [Phabricator](http://phabricator.org/) - Collection of web applications that help build better software. ([Demo](https://secure.phabricator.com/), [Source Code](https://github.com/phacility/phabricator)) `Apache` `PHP`
   * [Redmine](http://www.redmine.org/) - Redmine is a flexible project management web application. `GPLv2` `Ruby`
   * [Taiga](https://taiga.io/) - Agile Project Management Tool based on the Kanban and Scrum methods.  ([Source Code](https://github.com/taigaio))
+  * [TFS 2016](https://www.visualstudio.com/downloads/download-visual-studio-vs) - Team Foundation Server (TFS) Express 2015 Update 3 is a free, source code-control (powered by Git or TFVC) project management and team collaboration platform for individual developers and small teams of five or less.
```

Comment:
From the research I've just done, it looks like it's free for up to 5 users after which you need to license it. So it would belong in Non-Free because of that at the very least.

You've labelled it as TFS 2016, it's 2015.